pkg/splunk/result_test.go

```diff
@@ -0,0 +1,70 @@
+package splunk
+
+import (
+	"reflect"
+	"testing"
+	"time"
+)
+
+func TestSearchResult_string(t *testing.T) {
+	type args struct {
+		field string
+	}
+	tests := []struct {
+		name string
+		a    SearchResult
+		args args
+		want string
+	}{
+		// TODO: Add test cases.
```

Comment:
@karthikperu7 - should I create another card in the same epic for implementing tests using the framework Grant wrote here?  Or should that be another milestone?